Sources/Extensions/Foundation/DateExtensions.swift

```diff
@@ -457,6 +457,15 @@ public extension Date {
         return Calendar.current.timeZone
     }
 
+    /// SwifterSwift: Yesterday date.
+    ///
+    ///     var date = Date() // "Oct 3, 2018, 10:57:11"
+    ///     var yesterday = yesterday // "Oct 2, 2018, 10:57:11"
+    ///
+    public var yesterday: Date {
+        return self.addingTimeInterval(-86400.0)
```

Comment:
Actually if the whole purpose of this is to be a shorthand for `adding(_:value:)` why don't you use that?

Reply:
In the Guidelines it says "Do not use an existing SwifterSwift extension inside another SwifterSwift extension" so I didn't want to use `adding(_:value:)`